packages/dialog/test/dialog.common.js

```diff
@@ -64,6 +64,12 @@ describe('vaadin-dialog', () => {
         expect(overlay.getAttribute('role')).to.be.eql('dialog');
       });
 
+      it('should change role attribute on the overlay based on overlayRole', async () => {
```

Comment:
**nit:** Could be a snapshot test?

Reply:
Good point, thanks. Moved this test and removed two others for default `role` and `theme` as they are also covered by existing snapshot tests.

Reply:
Extracted changes to unrelated tests to #7586 so that this PR doesn't touch unit tests.